rpc/transport/connection.go

```diff
@@ -12,9 +12,9 @@ type Requester interface {
 	// Close closes the connection
 	Close()
 
-	// Request sends data for a topic and returns the response data or an error
+	// Request sends request data and returns the response data or an error
 	Request([]byte) ([]byte, error)
-	// Subscribe listens for notifications for a topic.
+	// Subscribe provides a notification channel.
 	// If subscription to a notification topic fails, it returns an error.
```

Comment:
Dangling mention of "topic" here?

Reply:
Good catch. Fixed in [19bbb43](https://github.com/statechannels/go-nitro/pull/1126/commits/19bbb43d75812a6bb9262f8b5be1946c27bab0b8)